gnomad/utils/sparse_mt.py

```diff
@@ -195,6 +197,12 @@ def _get_info_agg_expr(
         array.
     :param prefix: Optional prefix for the fields. Used for adding 'AS_' in the AS case.
     :param treat_fields_as_allele_specific: Treat info fields as allele-specific. Defaults to False.
+    :param retain_cdfs: If True, retains the cumulative distribution functions (CDFs) as an annotation
+        for `median_agg_fields`. Keeping the CDFs is useful for annotations that require calculating the median
+        across combined datasets at a later stage. Default is False.
+    :param cdf_k: Parameter controlling the accuracy vs. memory usage tradeoff when retaining CDFs. A higher
+        value of `cdf_k` results in a more accurate CDF approximation but increases memory usage and computation
+        time. Default is 200.
```

Comment:
Sorry, please excuse the OCD

```suggestion
    :param treat_fields_as_allele_specific: Treat info fields as allele-specific. 
        Defaults to False.
    :param retain_cdfs: If True, retains the cumulative distribution functions (CDFs) 
        as an annotation for `median_agg_fields`. Keeping the CDFs is useful for 
        annotations that require calculating the median across combined datasets at a 
        later stage. Default is False.
    :param cdf_k: Parameter controlling the accuracy vs. memory usage tradeoff when 
        retaining CDFs. A higher value of `cdf_k` results in a more accurate CDF 
        approximation but increases memory usage and computation time. Default is 200.
```